spec/migrations/20160214115800_inline_ems_id_spec.rb

```diff
@@ -0,0 +1,48 @@
+require_migration
+
+describe InlineEmsId do
+  let(:container_stub) { migration_stub(:Container) }
+  let(:container_group_stub) { migration_stub(:ContainerGroup) }
+  let(:container_definition_stub) { migration_stub(:ContainerDefinition) }
+  let(:container_group) { container_group_stub.create!(:ems_id => 23) }
+
+  migration_context :up do
+    it 'it adds a column called :ems_id to the containers table' do
```

Comment:
No need to test functions of Rails...only the data migration part needs to be tested.